test/e2e/test_tools.py

```diff
@@ -176,38 +176,45 @@ def test_create_ontology(self) -> None:
                         dump=False)
 
     def test_xml_upload(self) -> None:
-        xml_upload(input_file=self.test_data_file,
-                   server=self.server,
-                   user=self.user,
-                   password=self.password,
-                   imgdir=self.imgdir,
-                   sipi=self.sipi,
-                   verbose=False,
-                   validate_only=False,
-                   incremental=False)
+        with self.assertRaises(SystemExit) as cm:
+            xml_upload(input_file=self.test_data_file,
+                       server=self.server,
+                       user=self.user,
+                       password=self.password,
+                       imgdir=self.imgdir,
+                       sipi=self.sipi,
+                       verbose=False,
+                       validate_only=False,
+                       incremental=False)
+        self.assertEqual(cm.exception.code, 0)
 
         mapping_file = ''
         for mapping in [x for x in os.scandir('.') if x.name.startswith('id2iri_test-data_mapping_')]:
             delta = datetime.datetime.now() - datetime.datetime.fromtimestamp(mapping.stat().st_mtime_ns / 1000000000)
             if delta.seconds < 15:
                 mapping_file = mapping.name
```

Comment:
When would you have a file that starts with "id2iri_test-data_mapping_" which is older than 15 seconds?